registry/handlers/images.go

```diff
@@ -83,6 +91,64 @@ func (imh *imageManifestHandler) GetImageManifest(w http.ResponseWriter, r *http
 		return
 	}
 
+	supportsSchema2 := false
```

Comment:
Discussed offline, but noting here to keep the discussion up to date that the if/else statements below only run if `imh.Tag != ""`.